### PR TITLE
added hw-26-1

### PR DESCRIPTION
**Pull request by kenu21**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        }
+        return item;
+    }
+
+    @Override
+    public List<Item> getAllItems() {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            List<Item> list;
+            list = session.createCriteria(Item.class).list();
+            return list;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java]: return session.createCriteria(Item.class).list();

File: src/main/java/mate/academy/internetshop/model/Item.java [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+
+@Entity
+@Table(name = "items")
 public class Item {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "item_id")
     private Long id;
     private String name;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/model/Item.java:18]: optional: better add annotation column over all fields

**Review by Sofasmile — APPROVED** [review]
make small fixes

File: src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java (hunk 1 above)
Comment (kenu21) [reply src/main/java/mate/academy/internetshop/dao/hibernate/ItemDaoHibernateImpl.java]: OK

**Pull request merged**